docker/package/tezos_baking_wizard.py

```diff
@@ -0,0 +1,466 @@
+            print("Its address is ", address.decode("utf-8"))
+
+            replace = input("Should it be replaced with the new import? <y/N> ").strip()
+            if replace.lower() in ["y", "yes"]:
+                proc_call(cmd + " --force")
+
+
+    # Check if there is already some blockchain data in the tezos-node data directory,
+    # and ask the user if it can be overwritten.
+    def check_blockchain_data(self, snapshot_file):
+        node_dir = "/var/lib/tezos/node-" + self.config["network"]
+        node_dir_contents = os.listdir(node_dir)
+        clean = ["config.json", "version.json"]
+        diff = set(node_dir_contents) - set(clean)
+        if diff:
+            print("The Tezos node data directory already has some blockchain data:")
+            print("\n".join(diff))
+            print()
+            replace = input("Delete this data and import a new snapshot? <y/N> ").strip()
+            if replace.lower() in ["y", "yes"]:
+                for path in diff:
+                    try:
+                        os.remove(os.path.join(node_dir, path))
+                    except IsADirectoryError:
+                        shutil.rmtree(os.path.join(node_dir, path))
+
+                print("Node directory cleaned.")
+
+                return True
+
+        return False
+
+
+    # Importing the snapshot for Node bootstrapping
+    def import_snapshot(self):
+        snapshot_file = ""
+        if self.config["snapshot"] == "skip":
+            return
+        elif self.config["snapshot"] == "file":
+            snapshot_file = self.config["snapshot_file"]
+        elif self.config["snapshot"] == "url":
+            url = self.config["snapshot_url"]
+            snapshot_file = fetch_snapshot(url)
+        elif self.config["snapshot"] == "download rolling":
+            url = "https://" + self.config["network"] + ".xtz-shots.io/rolling"
+            snapshot_file = fetch_snapshot(url)
+        elif self.config["snapshot"] == "download full":
+            url = "https://" + self.config["network"] + ".xtz-shots.io/full"
+            snapshot_file = fetch_snapshot(url)
+
+        do_import = self.check_blockchain_data(snapshot_file)
+
+        if do_import:
+            proc_call(
+                "sudo -u tezos tezos-node-"
+                + self.config["network"]
+                + " snapshot import "
+                + snapshot_file
+            )
+
+            print("Snapshot imported.")
+
+    # Bootstrapping tezos-node
+    def bootstrap_node(self):
+
+        self.import_snapshot()
+
+        proc_call(
+            "sudo systemctl start tezos-node-" + self.config["network"] + ".service"
+        )
+
+        proc_call("sudo -u tezos tezos-client bootstrapped")
+
+        print()
+        print("The Tezos node bootstrapped successfully.")
+
+    # Importing the baker key
+    def import_baker_key(self):
+        if self.config["key_import_mode"] == "secret-key":
+            check_baker_account(
```

Comment:
I believe, `self` is missed here
```suggestion
            self.check_baker_account(
```